### PR TITLE
Creates JobStatus element.

### LINKED ISSUE #202: JobStatus element
Represents the status of a given Job. For example, if a job is cancelled this element would represent that status and or change of status.

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
Draft of elements to represents status of Site and Job. Closes #202, #200 .

### CODE REVIEW

**Comment by cwhanse** [comment]
I'm OK with the element names, a little puzzled by the "\" characters in the sample values. Are we thinking to create enumerations for these?

**Comment by kevinkirkl3y** [comment]
@cwhanse That was my bad. I fixed the sample values. I do think we may want to generate an enumerated list for these values but I wasn't sure if we wanted to create a specific Item type in order to that.

**Review by jrippingale — COMMENTED** [review]
Let's remove Site Status and add JobStatus with enumerated JobStatusItemType

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -17684,6 +17704,43 @@
...
+        "Cancelled": {
+          "label": "Cancelled",
+          "description": ""
+        },
+        "Completed": {
+          "label": "Completed",
+          "description": ""
+        },
+        "SalesPostponed": {
+          "label": "Sales-Postponed\t",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:17732]: delete tab character
```suggestion
          "label": "Sales-Postponed",
```

**Review by cwhanse — APPROVED** [review]

**Comment by kevinkirkl3y** [comment]
Rename Sales-Postponed to Sales-Lost

**Review by jrippingale — CHANGES_REQUESTED** [review]
Let's remove Sales-Postponed and Sales-Closed
rename Lost to Sales-Lost

**Review by jrippingale — APPROVED** [review]

**Comment by cwhanse** [comment]
@kevinkirkl3y ready to merge after dealing with the inevitable conflicts

**Review by jrippingale — APPROVED** [review]

**Pull request merged**